src/main/webapp/about.jsp

```diff
@@ -1,1570 +1,2132 @@
-                </tr>
-                <tr>
-                    <td>Aishwarya Seth</td>
-                    <td><a href="https://github.com/TEAMMATES/teammates/issues?q=involves%3AThe-Lady"
-                        target="_blank" rel="noopener noreferrer">bug reporting/fixing, enhancements</a></td>
-                </tr>
-                <tr>
-                    <td>Akshay Vijayvergia</td>
-                    <td><a href="https://github.com/TEAMMATES/teammates/issues?q=involves%3Aslidego"
-                        target="_blank" rel="noopener noreferrer">bug reporting/fixing, enhancements</a></td>
-                </tr>
-                <tr>
-                    <td>Alexander Levy</td>
-                    <td><a href="https://github.com/TEAMMATES/teammates/issues?q=involves%3Aallevy3"
-                        target="_blank" rel="noopener noreferrer">bug reporting/fixing, enhancements</a></td>
-                </tr>
-                <tr>
-                    <td>Alexandr Kolymago</td>
-                    <td><a href="https://github.com/TEAMMATES/teammates/issues?q=involves%3Ajusttimki"
-                        target="_blank" rel="noopener noreferrer">bug reporting/fixing, enhancements</a></td>
-                </tr>
-                <tr>
-                    <td>Aman Dhaliwal</td>
-                    <td><a href="https://github.com/TEAMMATES/teammates/issues?q=involves%3Aa-dhaliwal"
-                        target="_blank" rel="noopener noreferrer">bug reporting/fixing, enhancements</a></td>
-                </tr>
-                <tr>
-                    <td>Andy Daehn</td>
-                    <td><a href="https://github.com/TEAMMATES/teammates/issues?q=involves%3Aandydaehn"
-                        target="_blank" rel="noopener noreferrer">bug reporting/fixing, enhancements</a></td>
-                </tr>
-                <tr>
-                    <td>Andrew Hunter</td>
-                    <td><a href="https://github.com/TEAMMATES/teammates/issues?q=involves%3Aiamhunter"
-                        target="_blank" rel="noopener noreferrer">bug reporting/fixing, enhancements</a></td>
-                </tr>
-                <tr>
-                    <td>Angie Jones</td>
-                    <td><a href="https://github.com/TEAMMATES/teammates/issues?q=involves%3Aangiejones"
-                        target="_blank" rel="noopener noreferrer">bug reporting/fixing, enhancements</a></td>
-                </tr>
-                <tr>
-                    <td>Anirudh Jain</td>
-                    <td><a href="https://github.com/TEAMMATES/teammates/issues?q=involves%3Asponde25"
-                        target="_blank" rel="noopener noreferrer">bug reporting/fixing, enhancements</a></td>
-                </tr>
-                <tr>
-                    <td>Anthony Schneider</td>
-                    <td><a href="https://github.com/TEAMMATES/teammates/issues?q=involves%3ABr35Ba56"
-                        target="_blank" rel="noop
```

Comment:
Is it time to bring up JSP linting? Putting up your comment here for reference: https://github.com/TEAMMATES/teammates/issues/7499#issuecomment-307691511